backend/src/services/user/permissionChecker.ts

```diff
@@ -33,6 +33,24 @@ export default class PermissionChecker {
     }
   }
 
+  /**
+   * Validates if the user has any permission among specified
+   * and throws Error403 if it doesn't
+   */
+  validateHasAny(permissions) {
```

Comment:
**Nitpick**: we could try to use the `.some` method like we're using in other functions within this file.
Something like 
```javascript
if (!permissions.some((perm) => this.has(perm)) {
    throw new Error403(this.language)
}
```

Reply:
Done - nice one - didn't know Javascript had this method :)